### ISSUE TITLE
kedro-viz hook error when running Jupyter kedro magic

### ISSUE THREAD

**Issue by kennyb-qb** [kedro-org/kedro-viz#1816]
## Description
using kedro line magic produces error "Plugin already registered under a different name"
work around is to use settings.py `DISABLE_HOOKS_FOR_PLUGINS` or uninstall kedro-viz

## Context
unable to load kedro ipython magic by default without extra settings or uninstalling kedro-viz

## Steps to Reproduce

1. linux docker environment, python 3.9, pip install dependencies
2. create new notebook and run kedro magic as per doc: https://docs.kedro.org/en/stable/notebooks_and_ipython/kedro_and_notebooks.html#loading-the-project-with-the-kedro-ipython-extension

## Expected Result
Kedro magic loads the project and global variables

## Actual Result
`Plugin already registered under a different name`
![image](https://github.com/kedro-org/kedro-viz/assets/101136683/4fa861fd-e28f-4668-b54b-74618a0aa918)

## Your Environment
Include as many relevant details as possible about the environment you experienced the bug in:

* Web browser system and version: google chrome
* Operating system and version: linux docker image on Domino data lab
* NodeJS version used (if relevant): node@16.5.0 (client requires this downgrade)
* Kedro version used (if relevant): 0.18.14
* Python version used (if relevant): python 3.9
pip freeze:
absl-py==0.8.0
aiobotocore==2.1.2
aiofiles==22.1.0
aiohttp==3.8.4
aioitertools==0.11.0
aiosignal==1.3.1
aiosqlite==0.19.0
alembic==1.10.4
antlr4-python3-runtime==4.9.3
anyconfig==0.14.0
anyio==3.6.2
aplus==0.11.0
appdirs==1.4.4
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
asn1crypto==1.5.1
astropy==6.0.0
astropy-iers-data==0.2024.1.29.0.30.37
asttokens==2.2.1
async-timeout==4.0.2
attrs==21.4.0
Automat==22.10.0
Babel==2.12.1
backcall==0.2.0
backoff==1.11.1
beautifulsoup4==4.12.2
binaryornot==0.4.4
black==22.12.0
blake3==0.3.3
bleach==6.0.0
blinker==1.7.0
blosc2==2.5.1
boto3==1.20.23
botocore==1.23.24
bowler==0.9.0
bqplot==0.12.42
branca==0.7.1
Brotli==1.0.9
brotlipy==0.7.0
bson==0.5.10
build==1.1.1
cachetools==5.3.0
cairocffi==1.4.0
certifi==2022.12.7
cffi @ file:///croot/cffi_1670423208954/work
cfgv==3.4.0
chardet==3.0.4
charset-normalizer==2.1.1
click==8.1.7
cloudpickle==2.2.1
colorama==0.4.6
comm==0.1.3
compress-pickle==2.1.0
conda==4.14.0
conda-package-handling @ file:///croot/conda-package-handling_1672865015732/work
conda_package_streaming @ file:///croot/conda-package-streaming_1670508151586/work
constantly==15.1.0
contourpy==1.0.7
cookiecutter==2.6.0
coverage==7.4.4
cryptography==36.0.2
cssselect==1.2.0
cx-Oracle==8.3.0
cycler==0.11.0
dash==1.16.2
dash-bootstrap-components==0.10.7
dash-core-components==1.12.1
dash-daq==0.5.0
dash-html-components==1.1.1
dash-renderer==1.8.2
dash-table==4.10.1
dask==2023.4.1
databricks-cli==0.17.6
db-dtypes==1.1.1
debugpy==1.6.7
decorator==5.1.1
defusedxml==0.7.1
delta-spark==2.3.0
Deprecated==1.2.13
dill==0.3.6
distlib==0.3.8
dnspython==2.5.0
docker==6.0.1
docopt==0.6.2
domino-code-assist==1.1.4
domino-data-capture==0.1.2
dominodatalab==1.2.3
dominodatalab-data==5.6.0
dynaconf==3.2.5
entrypoints==0.4
et-xmlfile==1.1.0
exceptiongroup==1.2.0
executing==1.2.0
fastapi==0.110.0
fastavro==1.7.3
fastjsonschema==2.16.3
feast==0.29.0
filelock==3.13.2
fissix==21.11.13
Flask==3.0.1
Flask-Compress==1.13
Flask-Cors==3.0.10
flask-marshmallow==1.1.0
Flask-Migrate==4.0.5
Flask-OpenTracing==1.1.0
Flask-SQLAlchemy==3.1.1
flatbuffers==1.12
fonttools==4.39.3
fqdn==1.5.1
frozendict==2.3.8
frozenlist==1.3.3
fsspec==2022.1.0
future==0.18.3
gitdb==4.0.10
GitPython==3.1.31
google-api-core==2.11.0
google-auth==2.17.3
google-cloud-bigquery==3.10.0
google-cloud-bigquery-storage==2.19.1
google-cloud-bigtable==2.17.0
google-cloud-core==2.3.2
google-cloud-datastore==2.15.1
google-cloud-storage==2.8.0
google-crc32c==1.5.0
google-resumable-media==2.5.0
googleapis-common-protos==1.59.0
graphql-core==3.2.3
greenlet==2.0.2
grpc-google-iam-v1==0.12.6
grpcio==1.54.0
grpcio-opentracing==1.1.4
grpcio-reflection==1.34.1
grpcio-status==1.54.0
gunicorn==20.1.0
h11==0.14.0
h5py==3.10.0
hdfs==2.7.3
hiredis==2.2.2
httpcore==0.16.3
httptools==0.5.0
httpx==0.23.3
humanize==4.6.0
hyperlink==21.0.0
identify==2.5.35
idna==3.3
importlib-metadata==6.6.0
importlib_resources==6.4.0
impyla==0.19.0
incremental==22.10.0
iniconfig==2.0.0
ipydatawidgets==4.3.5
ipykernel==6.19.4
ipyleaflet==0.18.2
ipympl==0.9.3
ipython==8.13.1
ipython-genutils==0.2.0
ipyvolume==0.6.3
ipyvue==1.9.0
ipyvuetify==1.8.10
ipywebrtc==0.6.0
ipywidgets==7.7.3
isoduration==20.11.0
itemadapter==0.8.0
itemloaders==1.1.0
itsdangerous==2.1.2
jaeger-client==4.4.0
jedi==0.18.2
Jinja2==3.1.3
jmespath==0.10.0
joblib==1.2.0
json5==0.9.11
jsonify==0.5
jsonpointer==2.3
jsonschema==4.17.3
jupyter==1.0.0
jupyter-console==6.6.3
jupyter-events==0.6.3
jupyter-server-mathjax==0.2.6
jupyter-ydoc==0.2.4
jupyter_client==8.2.0
jupyter_core==5.3.0
jupyter_server==2.5.0
jupyter_server_fileid==0.9.0
jupyter_server_proxy==4.0.0
jupyter_server_terminals==0.4.4
jupyter_server_ydoc==0.8.0
jupyterlab==3.6.3
jupyterlab-git==0.41.0
jupyterlab-pygments==0.2.2
jupyterlab-widgets==1.1.4
jupyterlab_server==2.22.1
kedro==0.18.14
kedro-datasets==1.8.0
kedro-telemetry==0.2.5
kedro-viz==6.5.0
Keras==2.4.3
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
kiwisolver==1.4.4
lazy_loader==0.3
lightgbm==4.3.0
llvmlite==0.40.0
locket==1.0.0
loguru==0.5.3
lxml==4.9.2
lz4==4.3.3
Mako==1.2.4
Markdown==3.4.3
markdown-it-py==2.2.0
MarkupSafe==2.1.2
marshmallow==3.20.2
marshmallow-sqlalchemy==1.0.0
matplotlib==3.6.2
matplotlib-inline==0.1.6
mdit-py-plugins==0.3.5
mdurl==0.1.2
mistune==2.0.5
mlflow==2.2.2
mmh3==3.1.0
more-itertools==10.2.0
moreorless==0.4.0
msgpack==1.0.8
multidict==6.0.4
mypy==1.2.0
mypy-extensions==1.0.0
mysql-connector==2.2.9
mysql-connector-python==8.3.0
mysqlclient==2.2.1
nbclassic==0.5.6
nbclient==0.7.4
nbconvert==7.3.1
nbdime==3.2.1
nbformat==5.8.0
ndindex==1.8
nest-asyncio==1.5.6
nltk==3.4.5
nodeenv==1.8.0
notebook==6.5.4
notebook_shim==0.2.3
numba==0.57.0
numexpr==2.9.0
numpy==1.23.3
oauthlib==3.2.2
omegaconf==2.3.0
openpyxl==3.1.2
opentracing==2.4.0
orjson==3.9.15
oscrypto==1.3.0
packaging==23.2
pandas==1.5.0
pandas-gbq==0.17.9
pandavro==1.5.2
pandocfilters==1.5.0
parse==1.20.1
parsel==1.8.1
parso==0.8.3
partd==1.4.0
pathspec==0.12.1
patsy==0.5.6
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.5.0
pip-tools==7.4.1
platformdirs==4.2.0
plotly==5.10.0
plotly-express==0.4.1
pluggy==1.2.0
polling2==0.5.0
pre-commit==3.6.2
progressbar2==4.2.0
prometheus-client==0.8.0
prompt-toolkit==3.0.38
Protego==0.2.1
proto-plus==1.22.2
protobuf==4.25.3
psutil==5.9.5
ptyprocess==0.7.0
pure-eval==0.2.2
pure-sasl==0.6.2
py-cpuinfo==9.0.0
py4j==0.10.9.5
pyarrow==15.0.2
pyasn1==0.5.0
pyasn1-modules==0.3.0
pycosat @ file:///croot/pycosat_1666805502580/work
pycparser @ file:///tmp/build/80754af9/pycparser_1636541352034/work
pycryptodome==3.17
pycryptodomex==3.15.0
pydantic==1.10.7
pydata-google-auth==1.8.2
PyDispatcher==2.0.7
pyerfa==2.0.1.1
Pygments==2.15.1
pyhdb==0.3.4
PyJWT==2.4.0
pymdown-extensions==9.11
pymongo==4.6.1
pymssql==2.2.11
pyOpenSSL==22.0.0
pyparsing==3.0.9
pyproject_hooks==1.0.0
pyrsistent==0.19.3
PySocks @ file:///tmp/build/80754af9/pysocks_1605305812635/work
pyspark==3.3.2
pytest==7.4.4
pytest-cov==3.0.0
pytest-mock==1.13.0
python-dateutil==2.8.2
python-dotenv==1.0.0
python-json-logger==2.0.7
python-slugify==8.0.4
python-utils==3.5.2
pythonping==1.1.4
pythreejs==2.4.2
pytoolconfig==1.3.1
pytz==2022.2.1
PyWavelets==1.4.1
PyYAML==6.0
pyzmq==25.0.2
qtconsole==5.4.2
QtPy==2.3.1
querystring-parser==1.2.4
queuelib==1.6.2
reacton==1.5.1
redis==4.2.2
redshift-connector==2.1.0
requests==2.28.1
requests-file==1.5.1
rfc3339-validator==0.1.4
rfc3986==1.5.0
rfc3986-validator==0.1.1
rich==13.3.5
rich-click==1.6.1
rope==1.13.0
rsa==4.9
ruamel-yaml-conda @ file:///tmp/build/80754af9/ruamel_yaml_1616016711199/work
ruff==0.0.292
s3fs==0.4.2
s3transfer==0.5.2
scikit-learn==1.3.2
scipy==1.12.0
scramp==1.4.4
Scrapy==2.8.0
seaborn==0.13.2
secure==0.3.0
seldon-core==1.14.1
Send2Trash==1.8.2
service-identity==21.1.0
shap==0.45.0
simpervisor==0.4
simpy==4.1.1
six==1.16.0
slicer==0.0.7
smmap==5.0.0
sniffio==1.3.0
snowflake-connector-python==2.7.12
solara==1.10.0
solara-assets==1.10.0
soupsieve==2.4.1
SQLAlchemy==1.4.52
sqlalchemy-redshift==0.8.14
sqlalchemy2-stubs==0.0.2a34
sqlparse==0.4.4
stack-data==0.6.2
starlette==0.36.3
statsmodels==0.14.1
strawberry-graphql==0.221.1
tables==3.9.2
tabulate==0.9.0
tenacity==8.2.2
teradatasql==17.20.0.19
terminado==0.17.1
text-unidecode==1.3
threadloop==1.0.2
threadpoolctl==3.1.0
thrift==0.16.0
thrift-sasl==0.4.3
tinycss2==1.2.1
tldextract==3.4.1
toml==0.10.2
tomli==2.0.1
toolz @ file:///home/conda/feedstock_root/build_artifacts/toolz_1657485559105/work
toposort==1.10
tornado==6.3.1
tqdm==4.65.0
traitlets==5.9.0
traittypes==0.2.1
Twisted==22.10.0
typeguard==2.13.3
typing_extensions==4.3.0
uri-template==1.2.0
urllib3==1.26.12
uvicorn==0.27.1
uvloop==0.17.0
uWSGI @ file:///croot/uwsgi_1679988297904/work
vaex==4.17.0
vaex-astro==0.9.3
vaex-core==4.17.1
vaex-hdf5==0.14.1
vaex-jupyter==0.8.2
vaex-ml==0.18.3
vaex-server==0.9.0
vaex-viz==0.5.4
virtualenv==20.25.1
volatile==2.1.0
w3lib==2.1.1
watchdog==3.0.0
watchfiles==0.19.0
watchgod==0.8.2
wcwidth==0.2.6
webcolors==1.13
webencodings==0.5.1
websocket-client==1.5.1
websockets==11.0.2
Werkzeug==3.0.1
widgetsnbextension==3.6.4
wrapt==1.15.0
xarray==2024.1.1
xgboost==2.0.3
xlrd==2.0.1
XlsxWriter==3.1.9
xyzservices==2023.10.1
y-py==0.5.9
yarl==1.9.2
ypy-websocket==0.8.2
zipp==3.15.0
zope.interface==6.0
zstandard @ file:///croot/zstandard_1677013143055/work

## Checklist
- [ x] Include labels so that we can categorise your issue

**Comment by ravi-kumar-pilla** [comment]
Hi @kennyb-qb ,

The hook name was changed in Kedro-Viz v6.6.1 [here](https://github.com/kedro-org/kedro-viz/pull/1544). I see your kedro-viz version as v6.5.0 from `pip freeze`. 

I am not sure how this is affecting the kedro magic line in your notebook (somehow the notebook instance has a kedro-viz version >=6.6.1) . Could you please upgrade the kedro-viz version to `>=6.6.1` in your local and try this out ? 

Thank you

**Comment by ravi-kumar-pilla** [comment]
I spoke to @kennyb-qb and it seems his teammate had some issues with the local environment. This is not an issue anymore and he suggested to close it for now. Thank you
